sdk/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpanProcessor.java

```diff
@@ -77,8 +78,8 @@ public void onEnd(ReadableSpan span) {
     }
     try {
       List<SpanData> spans = Collections.singletonList(span.toSpanData());
-      spanExporter.export(spans);
-    } catch (Throwable e) {
+      spanExporter.export(spans).get();
```

Comment:
ditto on needing an issue for this one as well.

Reply:
This will go away